content/en/user-manual/publishing/playable-ads/fb-playable-ads.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Facebook Playable Ad
+template: usermanual-page.tmpl.html
+position: 3
+---
+
+PlayCanvas supports the [Facebook Playable Ad][1] formats and requirements via an [official external tool on GitHub][2].
+
+<video loop controls autoplay muted src="/videos/fb-playable-ad-bitmoji-creator.mp4" width="200"> </video> <video loop controls autoplay muted src="/videos/fb-playable-ad-cube-jump.mp4" width="200"></video> <video loop controls autoplay muted src="/videos/fb-playable-ad-flappy-bird.mp4" width="200"></video>
+
+The tool can create both the single 2MB (uncompressed) HTML file and the 5MB (uncompressed) ZIP formats via the configuration options. Full specifications for Facebook Playable Ads can be found on their [help centre][3].
```

Comment:
Should this say `5MB (compressed)`?

Reply:
I'm under the impression that is uncompressed (the docs are really not clear) https://www.facebook.com/business/help/412951382532338?ref=FBB_PlayableAds

```
The total size of uploaded files may not exceed 5 MB.
The number of files inside the bundle may not exceed 100.
The size of index.html file should not exceed 2 MB.
```